build/yamls/base/security-crds.yml

```diff
@@ -43,12 +43,10 @@ spec:
                properties:
                  podSelector:
                    type: object
-                   additionalProperties:
-                     type: object
+                   x-kubernetes-preserve-unknown-fields: true
```

Comment:
I believe this is correct. However, do you know why we specify `type: object` here, but not for the `podSelector` and `namespaceSelector` fields under `ingress` and `egress`?
This is what we have for `ingress` for example:
```
                     properties:
                       podSelector:
                         x-kubernetes-preserve-unknown-fields: true
                       namespaceSelector:
                         x-kubernetes-preserve-unknown-fields: true
```